ViewModels/BynderFieldDisplayViewModel.cs

```diff
@@ -1,9 +1,20 @@
 using CSM.Bynder.Fields;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace CSM.Bynder.ViewModels;
 
 public class BynderFieldDisplayViewModel
 {
     public IList<BynderResource> Resources { get; } = new List<BynderResource>();
+
+    public Task AddResourcesAsync(IList<BynderResource> resources)

```

Comment:
Is there any reason that this is `async`? Also, in the other places. I see no reason for this being `async`, but let me know.

Reply:
Analyser wants it to be called async I was surprised aswell.

Reply:
I just replaced some of them and there are no warnings. The problem should be at your end.

Reply:
I can make them void tho and that way it is resolved. In the Filters module I made the methods void because making them Task would result in a circular warning generation:
If Task use Async but the in place where it used it was suggested to use await coz it's a Task but then the whole method suggested don't use async await at all and so on.